docs/data-management-on-carbondata.md

```diff
@@ -703,6 +704,194 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
   * The partitioned column can be excluded from SORT_COLUMNS, this will let other columns to do the efficient sorting.
   * When writing SQL on a partition table, try to use filters on the partition column.
 
+## PRE-AGGREGATE TABLES
```

Comment:
Please add some example to show the plan matching mechanism, like what query will hit which datamap

Reply:
added example